### ISSUE TITLE
CLI params not accepted when config.json is in path

### ISSUE THREAD

**Issue by tmeinlschmidt** [blockwatch-cc/tzindex#48]
Hi,
using latest tzindex 

    INFO MAIN Blockwatch Data Inc. TzIndex -- v15.0.0 49e3e0e

When I specify cmd params like

    tzindex -db.path=/data/index -rpc.url=https://...

it ended up like

```
2023/02/20 08:05:25.320981 INFO BLOC Registered custom metadata extension.
2023/02/20 08:05:25.321011 INFO MAIN Using bolt database ./db/xtz
```

so `db.path` (and any other parameter) is not honored. When I remove config.json, it works. I expected that CLI params overwrite config loaded from config.json (or defaults).

**Comment by echa** [comment]
Thanks for reporting. Fixed in commit 54ca882 This was an oversight when migrating to the new config system. We've been using ENV vars in our infra setup which did not have this problem.

New priority order is `default < config file < CLI args < ENV vars`

**Comment by tmeinlschmidt** [comment]
Thanks!
